tests/scalers/rabbitmq-queue-amqp.test.ts

```diff
@@ -35,6 +35,13 @@ test.serial(`Deployment should scale to 4 with ${messageCount} messages on the q
   t.true(await waitForDeploymentReplicaCount(0, 'test-deployment', testNamespace, 50, 5000), 'Replica count should be 0 after 3 minutes')
 })
 
+test.serial(`Deployment shouldn't scale because the 5 messages on the queue are less than ${minMetricValue}(minMetricValue)`, async t => {
+  RabbitMQHelper.publishMessages(t, testNamespace, connectionString, 5, queueName)
+
+  // with messages published, the consumer deployment should start receiving the messages
+  t.true(await waitForDeploymentReplicaCount(0, 'test-deployment', testNamespace, 20, 5000), 'Replica count remain 0 after 10 seconds')
+})
```

Comment:
I'm not 100 sure about this test. If the workload is already scaled to zero, `waitForDeploymentReplicaCount` will automatically detect 0 and will pass it without checking anything. Maybe if we introduce a sleep here for IDK, 60 seconds and then check the instances, we can check that they haven't scaled out